style.css

```diff
@@ -89,3 +89,253 @@
 .social li {
   margin: 10px;
 }
+
+/* work section */
+
+.work-section {
+  display: flex;
+  flex-direction: column;
+  background: #e5e5e5;
+}
+
+.card-section {
+  display: flex;
+  flex-direction: column;
+  margin: 100px 24px 50px 24px;
+  background: #fff;
+  padding: 0;
+  justify-content: flex-start;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.profile {
+  margin: 16px 16px 0 16px;
+}
+
+.card-section h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 12px 16px 5px 16px;
+  display: flex;
+  text-align: left;
+  align-items: center;
+  color: #172b4d;
+}
+
+.canopy {
+  display: flex;
+  flex-direction: row;
+  margin: 12px 75px 0 10px;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 0;
+}
+
+.card-section .cano {
+  color: #172b4d;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 16px;
+}
+
+.canopy li {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 6px;
+  font-family: poppins, sans-serif;
+  color: #7a869a;
+}
```

Comment:
- [ ] Please 🙏🏽 ensure that the styles like font size and margins are similar to the Figma template, everything is in ONE line.

| Figma template | Yor design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/235265330-c12e4e60-5316-4045-a3b6-73fb86be8814.png) | ![image](https://user-images.githubusercontent.com/58642949/235265405-79d46662-69f6-4f58-be34-9a3bb07ac790.png) |